libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -615,7 +616,12 @@ bool AP_Arming::servo_checks(bool report) const
         check_passed = false;
     }
 #endif
-    
+
+    if (AP::internalerror().errors() != 0) {
+        check_failed(ARMING_CHECK_NONE, report, "Internal errors detected");
```

Comment:
Definitely need to print the hex value of the error on this, or helping any user who gets it is going to be hard without hounding them for logs.